predict.py

```diff
@@ -294,5 +334,30 @@ def main(args):
         action="store_true",
         help="Whether the structure is NMR. NMR will have different states so TIMED will try to build a consensus",
     )
+
+    parser.add_argument(
+        "--res_to_fix",
+        type=parse_residue_list,
+        help="Specify the residue numbers that need to be fixed, separated by commas. Example: '2 3 4 5, 6 7 8' will keep 2 3 4 5 6 7 8th residues as identical with WT and predict the rest.",
+    )
+
+    parser.add_argument(
+        "--chains_to_fix",
+        type=parse_chain_list,
+        help="Specify the chains need to be fixed, separated by commas. Example: 'A, B' ",
+    )
+
+    parser.add_argument(
+        "--chains_to_predict",
+        type=parse_chain_list,
+        help="Specify the chains need to be predicted, separated by commas. Example: 'A, B' ",
+    )
+
+    parser.add_argument(
+        "--res_to_predict",
+        type=parse_residue_list,
+        help="Specify the residue numbers that need to be predicted, separated by commas. Example: '2 3 4 5, 6 7 8' will predict 2 3 4 5 6 7 8th residues and leave the rest as same as WT",
+    )
+
```

Comment:
How is this different from res_to_fix?

In theory we should predict anything that is not res to fix. right?